src/examples/text-inputs/FauxInput.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Field, Label, FauxInput } from '@zendeskgarden/react-forms';
+
+const Example = () => (
+  <Field>
+    <Label>Plant</Label>
+    <FauxInput />
```

Comment:
Should this include content within the `FauxInput` to show the expected use?

Reply:
Text content has been added within the `FauxInput`.